db/migrations/20190509101525_create_report_schedule_table_migration.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Copyright (C) 2019 Xibo Signage Ltd
+ *
+ * Xibo - Digital Signage - http://www.xibo.org.uk
+ *
+ * This file is part of Xibo.
+ *
+ * Xibo is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * any later version.
+ *
+ * Xibo is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Xibo.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+use Phinx\Migration\AbstractMigration;
+
+/**
+ * Class CreateReportScheduleTableMigration
+ */
+class CreateReportScheduleTableMigration extends AbstractMigration
+{
+    /** @inheritdoc */
+    public function change()
+    {
+        $table = $this->table('reportschedule', ['id' => 'reportScheduleId']);
+        $table->addColumn('name', 'string')
+            ->addColumn('reportName', 'string')
+            ->addColumn('filterCriteria', 'text')
+            ->addColumn('schedule', 'string')
+            ->addColumn('lastRunDt', 'integer', ['default' => 0])
+            ->addColumn('userId', 'integer')
+            ->addColumn('createdDt', 'integer')
+            ->create();
```

Comment:
if we use `save()` here, then I think we can make use of rollback in some circumstances